sublime-phpunit.py

```diff
@@ -98,6 +102,18 @@ def run(self, *args, **kwargs):
 
         self.run_in_terminal('cd ' + phpunit_config_path + ' && phpunit ' + file_name + " --filter '/::" + current_function + "$/'")
 
+class RunLastPhpunitTestCommand(PhpunitTestCommand):
+
+    def run(self, *args, **kwargs):
+        file_name, phpunit_config_path, phpunit_bin, active_view, directory = self.get_paths()
+
+        current_function = self.get_current_function(active_view)
+
+        if 'Test' in file_name:
+            self.run_in_terminal('cd ' + phpunit_config_path + ' && phpunit ' + file_name + " --filter '/::" + current_function + "$/'")
```

Comment:
Can you reuse existing implementation, ie run the method inside the `RunSinglePhpunitTestCommand` class? Maybe extend it and call the parent's `run()` method?